docs/api/python/z3compat/arith.rst

```diff
@@ -1,2 +1,84 @@
 Arithmetic
 ============
+
+
+Basic Arithmetic Term Builders
+-------------------
+.. autofunction:: cvc5_z3py_compat.Int
+.. autofunction:: cvc5_z3py_compat.Real
+.. autofunction:: cvc5_z3py_compat.IntVal
+.. autofunction:: cvc5_z3py_compat.RealVal
+.. autofunction:: cvc5_z3py_compat.RatVal
+.. autofunction:: cvc5_z3py_compat.Q
+.. autofunction:: cvc5_z3py_compat.IntSort
+.. autofunction:: cvc5_z3py_compat.RealSort
+.. autofunction:: cvc5_z3py_compat.FreshInt
+.. autofunction:: cvc5_z3py_compat.Ints
+.. autofunction:: cvc5_z3py_compat.IntVector
+.. autofunction:: cvc5_z3py_compat.FreshReal
+.. autofunction:: cvc5_z3py_compat.Reals
+.. autofunction:: cvc5_z3py_compat.RealVector
+
+
+Arithmetic Operators
+-------------------
+.. autofunction:: cvc5_z3py_compat.ToReal
+.. autofunction:: cvc5_z3py_compat.ToInt
+.. autofunction:: cvc5_z3py_compat.IsInt
+.. autofunction:: cvc5_z3py_compat.Sqrt
+.. autofunction:: cvc5_z3py_compat.Cbrt
+
+See the following operator overloads for buildign other kinds of arithmetic
+terms:
+
+* :py:meth:`cvc5_z3py_compat.ArithRef.__add__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__sub__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__mul__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__div__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__pow__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__neg__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__gt__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__lt__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__ge__`
+* :py:meth:`cvc5_z3py_compat.ArithRef.__le__`
+* :py:meth:`cvc5_z3py_compat.ExprRef.__eq__`
+* :py:meth:`cvc5_z3py_compat.ExprRef.__ne__`
+
+Testers
+-------------------
+.. autofunction:: cvc5_z3py_compat.is_arith_sort
```

Comment:
This again looks a bit inconsistent. Why is there `is_arith_sort`, but not `is_real_sort`? Conversely, we have `is_int_value` but not `is_arith_value`. Why is `is_bool_value` here?

Reply:
I moved `is_bool_value`. Good catch!

It seems that these testing functions are designed to line up with the classes. I'm not really sure what to do with them. To be frank, I do not think they're very useful, but we can't remove the existing ones without breaking with Z3Py.